frontend/workflows-lib/lib/components/workflow/TasksTable.tsx

```diff
@@ -12,7 +12,7 @@ const TasksTable: React.FC<TaskTableProps> = ({ tasks }) => {
   return (
     <Grid container spacing={1.5} padding={2} sx={{ overflow: "auto" }}>
       {tasks.map((task) => (
-        <Grid item key={task.name} xs="auto">
+        <Grid item key={task.id} xs="auto">
```

Comment:
Do we want to use the unique id here too?

Reply:
Yup, it complains!